src/coffee/io/reader.coffee

```diff
@@ -6,18 +6,24 @@ iconv = require 'iconv-lite'
 fs = Promise.promisifyAll require('fs')
 Excel = require 'exceljs'
 
+#debugLog = console.log
+debugLog = _.noop
```

Comment:
For enabling log messages, we can switch it for `console.log`. If we put there `if-else` it will lower the coverage just because of an untested condition.